cli/command/run/cmd.go

```diff
@@ -346,6 +350,72 @@ func runPlaywrightInSauce(p playwright.Project, regio region.Region, creds *cred
 	return r.RunProject()
 }
 
+func runTestcafe(cmd *cobra.Command, cli *command.SauceCtlCli) (int, error) {
+	p, err := testcafe.FromFile(cfgFilePath)
+	if err != nil {
+		return 1, err
+	}
+	p.Sauce.Metadata.ExpandEnv()
+	applyDefaultValues(&p.Sauce)
+	overrideCliParameters(cmd, &p.Sauce)
+
+	for k, v := range env {
+		for _, s := range p.Suites {
+			if s.Env == nil {
+				s.Env = map[string]string{}
+			}
+			s.Env[k] = v
+		}
+	}
+
+	if showConsoleLog {
+		p.ShowConsoleLog = true
+	}
+
+	if cmd.Flags().Lookup("suite").Changed {
+		if err := filterTestcafeSuite(&p); err != nil {
+			return 1, err
+		}
+	}
+	creds := credentials.Get()
+	if creds == nil {
+		return 1, errors.New("no sauce credentails set")
+	}
+
+	regio := region.FromString(p.Sauce.Region)
+	if regio == region.None {
+		log.Error().Str("region", regionFlag).Msg("Unable to determine sauce region.")
+		return 1, errors.New("no sauce region set")
+	}
+
+	tc := testcomposer.Client{
+		HTTPClient:  &http.Client{Timeout: testComposerTimeout},
+		URL:         regio.APIBaseURL(),
+		Credentials: *creds,
+	}
+
+	switch testEnv {
+	case "docker":
+		return runTestcafeInDocker(p, cli, tc)
+	case "sauce":
```

Comment:
Remove this case then, that's what the `default` is for ;) Since this case is not supported, we shouldn't just silently do nothing.

Reply:
ok, gotcha